pkgs/os-specific/linux/nixos-rebuild/default.nix

```diff
@@ -1,14 +1,22 @@
-{ substituteAll, runtimeShell, jq, nix, lib }:
```

Comment:
I think you did some mistake when rebasing. Please drop the changes to nixos-rebuild.